File: yt/yt/flow/examples/java/url_downloader/url_downloader/src/main/java/tech/ytsaurus/flow/examples/urldownloader/model/UrlMessage.java
package tech.ytsaurus.flow.examples.urldownloader.model;

import javax.persistence.Entity;

// Field order must match the "urls" stream definition in the static spec.
@Entity
public class UrlMessage {
    private String host;

    private String url;

    public UrlMessage() {
    }

    public UrlMessage(String host, String url) {
        this.host = host;
        this.url = url;
    }

    public String getHost() {
        return host;
    }

    public void setHost(String host) {
        this.host = host;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }
}

File: yt/yt/flow/examples/kotlin/url_downloader/url_downloader/src/main/kotlin/tech/ytsaurus/flow/examples/urldownloader/model/UrlMessage.kt
package tech.ytsaurus.flow.examples.urldownloader.model

import javax.persistence.Entity

// Field order must match the "urls" stream definition in the static spec.
@Entity
class UrlMessage {
    var host: String? = null

    var url: String? = null

    constructor()

    constructor(host: String?, url: String?) {
        this.host = host
        this.url = url
    }
}
